### PR TITLE
Update version to 1.1.0-alpha

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: code/main/core/configurationModule.brs [hunk 1]
```diff
@@ -26,7 +26,7 @@ function _adb_ConfigurationModule() as object
         _edge_configId: invalid,
         _edge_domain: invalid,
         ' media configuration
-        ' example: {"edgemedia.channel":"channel_x", "edgemedia.playerName": "player_y", "edgemedia.appVersion": "1.0.0"}
+        ' example: {"edgemedia.channel":"channel_x", "edgemedia.playerName": "player_y", "edgemedia.appVersion": "1.1.0-alpha"}
         _media_channel: invalid,
```
Comment (addb) [inline code/main/core/configurationModule.brs:30]: appVersion is nothing to do with SDK version.

File: sample/development/components/integration/test/integrationTests.brs [hunk 2]
```diff
@@ -25,7 +25,7 @@ function TS_SDK_integration() as object
             m.configId = test_config.config_id
             m.mediaChannel = "channel_test"
             m.mediaPlayerName = "player_test"
-            m.mediaAppVersion = "1.0.0"
+            m.mediaAppVersion = "1.1.0-alpha"
```
Comment (addb) [inline sample/development/components/integration/test/integrationTests.brs:28]: revert this change as mentioned in the previous comment.

File: sample/development/components/integration/test/integrationTests.brs [hunk 3]
```diff
@@ -521,7 +521,7 @@ function TS_SDK_integration() as object
                 ADB_assertTrue((debugInfo.configuration.edge_configid <> invalid and Len(debugInfo.configuration.edge_configid) > 10), LINE_NUM, "assert edge_configid is valid")
                 ADB_assertTrue((debugInfo.configuration.media_channel = "channel_test"), LINE_NUM, "assert media_channel is valid")
                 ADB_assertTrue((debugInfo.configuration.media_playerName = "player_test"), LINE_NUM, "assert media_playerName is valid")
-                ADB_assertTrue((debugInfo.configuration.media_appVersion = "1.0.0"), LINE_NUM, "assert media_appVersion is valid")
+                ADB_assertTrue((debugInfo.configuration.media_appVersion = "1.1.0-alpha"), LINE_NUM, "assert media_appVersion is valid")
```
Comment (addb) [inline sample/development/components/integration/test/integrationTests.brs:524]: can we have test-constant for this so we dont need to update the tests everywhere.

File: code/tests/edge/test_edgeRequestWorker.brs [hunk 4]
```diff
@@ -110,7 +110,7 @@ sub TC_adb_EdgeRequestWorker_processRequest_valid_response()
             implementationdetails: {
                 environment: "app",
                 name: "https://ns.adobe.com/experience/mobilesdk/roku",
-                version: "1.0.0"
+                version: "1.1.0-alpha"
```
Comment (addb) [inline code/tests/edge/test_edgeRequestWorker.brs:113]: can we add a constant for this.

File: sample/development/components/integration/test/integrationTests.brs (hunk 3 above)
Comment (yangyansong-adbe) [reply sample/development/components/integration/test/integrationTests.brs]: I prefer to keep those raw values in tests. It's easy to update those values when bumping the SDK version, and it's obvious when checking the test code.  Also, it's not straight forward to add a "constant" in Brightscript and to be referred to in multiple files.
